run-a-dv/integrations/dappnode.md

```diff
@@ -1,8 +1,8 @@
 # DappNode
 
-For setup, see quickstart guide:[​](https://docs.obol.org/next/run/integrations/Dappnode#for-setup-see-quickstart-guide)
+For set-up, see quickstart guide:[​](https://docs.obol.org/next/run/integrations/Dappnode#for-setup-see-quickstart-guide)
 
-For setup of a DV using Dappnode, see the quickstart guide [Create a DV Alone](https://docs.obol.org/next/run/start/quickstart_alone), and select the appropriate tab for "Dappnode".
+For set-up of a DV using DappNode, see the quickstart guide [Create a DV Alone](https://docs.obol.org/next/run/start/quickstart_alone), and select the appropriate tab for "DappNode".
```

Comment:
I think setup is more common than set-up?
Also Dappnode is officially spelled like Dappnode

```suggestion
For setup of a DV using Dappnode, see the quickstart guide [Create a DV Alone](https://docs.obol.org/next/run/start/quickstart_alone), and select the appropriate tab for "Dappnode".
```

Reply:
I think AI made DappNode looking identically to all other occurrences. As for setup vs set-up: noun vs verb as explained by AI, therefore different purpose.